bindings/matlab/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+find_program(
+	MATLAB_EXECUTABLE
+	NAMES matlab
+	PATHS ${Matlab_ROOT_DIR} ${Matlab_ROOT_DIR}/bin
```

Comment:
Yeah, I realized that which is why I mentioned we could just bundle the module instead of me copying more of its setup over, but I guess removing the line is fine for now.